monero-outreach-docs/translations/nl/quickfacts/quickfacts.md

```diff
@@ -51,47 +51,47 @@ Sinds de lancering heeft Monero verschillende aanzienlijke verbeteringen doorgem
 
 Het woord "Monero" komt van Esperanto. Men heeft deze taal gekozen omdat hij "gedecentraliseerd" is en symbool staat voor het breken van barrières tussen mensen wereldwijd. In Esperanto is "Monero" vrij opgebouwd uit drie elementen, elk één lettergreep: mon + er + o. Ieder heeft zijn eigen betekenis.
 
-**mon-:** geld
-
-**-er-:** kleinste deel
-
- **-o :** een ding / een zelfstandig naamwoord
++ **mon-:** geld
++
```

Comment:
@Josketobben, please remove the "+", this make it look like this:
![spaces](https://user-images.githubusercontent.com/7443480/55989037-f9475080-5c6a-11e9-9647-503924eefe66.png)

Just leave it without the "+".

Reply:
Oh it was just made, thanks. Closing this comment.